integration-tests/utils/aws/waitForDownloadHash.ts

```diff
@@ -1,6 +1,6 @@
 import { dynamoDBClient } from './dynamoDBClient'
 import { QueryCommand } from '@aws-sdk/client-dynamodb'
-import { getEnv } from '../helpers'
+import { getEnv, pause } from '../helpers'
 export const retrieveSecureDownloadDbRecord = async (
```

Comment:
Just one more thing, get rid of exports when you're not using them outside of the file. It makes things clearer when refactoring, because you then know exactly what's used where.

`export` is the equivalent of making a function `public` in Java. You don't need `retrieveSecureDownloadDbRecord` anywhere else than this file